client/state/memberships/gifts/actions.js

```diff
@@ -68,12 +43,6 @@ export const requestAddGift = ( siteId, gift, noticeText, onConfirm ) => {
 
 export const requestDeleteGift = ( siteId, giftId, noticeText ) => {
```

Comment:
Noting that some of this code is actually completely removable as it's not used, not just the non-referenced actions. Try looking for `requestDeleteGift` for example.

Reply:
> Noting that some of this code is actually completely removable as it's not used, not just the non-referenced actions. Try looking for `requestDeleteGift` for example.

Nice find, even more to remove! I double-checked all the updated action creators and I think it's just this one that's unreferenced currently. Updated in 97bc95f2199.